### ISSUE TITLE
Problem with reading 2D dataset - Scale vector of length 3 to 2d data

### ISSUE THREAD

**Issue by sebi06** [AllenCellModeling/napari-aicsimageio#54]
The bug is decribed here:

https://forum.image.sc/t/napari-napari-aicsimageio-faild-to-open-ome-tiff/68942?u=sebi06

**Comment by psobolewskiPhD** [comment]
This issue relates to the fact that the original data looks like this:
```
In [4]: img = aicsimageio.AICSImage(path)
In [5]: img.shape
Out[5]: (1, 1, 1, 2755, 3675)
```
the plugin squeezes the singletons and returns a 2D array.
But the metadata used for the scale info is:
```
In [9]: img.physical_pixel_sizes
Out[9]: PhysicalPixelSizes(Z=1.0, Y=0.227, X=0.227)
```

I think I have an idea how to fix, will try a PR.

**Comment by psobolewskiPhD** [comment]
Should be fixed with #60 which has been released.
Feel free to re-open if this pops up again.
